enable-kdump/cos-enable-kdump.yaml

```diff
@@ -0,0 +1,95 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Deploy this DaemonSet to enable kdump on the COS nodes with the
+# "cloud.google.com/gke-kdump-enabled=true" label.
+#
+# WARNING: Disabling hyper-threading might have severe performance impact on
+# your clusters and application. Please ensure that this is acceptable before
+# deploying this to your production clusters.
+#
+# WARNING: Enabling kdump requires node reboot. Therefore, in order to avoid
+# disrupting your workloads, it is recommended to create a new node pool with
+# with the "cloud.google.com/gke-kdump-enabled=true" label in your cluster,
+# deploy the DaemonSet to enable kdump in that node pool, and then migrate
+# your workloads to the new node pool.
+
```

Comment:
Please add something like this:

See https://cloud.google.com/kubernetes-engine/docs/tutorials/migrating-node-pool for how to migrate workloads between node pools.

Reply:
Yes. I will add documentation to here:
https://chromium.googlesource.com/chromiumos/overlays/board-overlays/+/refs/heads/master/overlay-lakitu/docs/kdump.md

I talked to the documentation people, and they are not happy about adding the user guide to cloud.google.com. So, COS team's repo it is.